src/redux/reducers/vendorReducer.js

```diff
@@ -1,17 +1,15 @@
 import { REGISTER_BUSINESS_SUCCESS } from '../actions/type';
 
 const initialState = {
-  status: null,
-  msg: ''
+  registrationDone: false
```

Comment:
@ritwizsinha This is not right method, server is returning status. Use that. 
Only on success the server is not returning a status code otherwise server is always responding with a status code.